packages/chai-a11y-axe/README.md

```diff
@@ -0,0 +1,86 @@
+# Chai a11y axe
```

Comment:
```suggestion
# Chai A11y aXe
```
either that or 
```md
# `chai-a11y-axe`
```

Reply:
should `A` be capital in `axe`? It doesn't feel right. I'm not native speaker though.

Reply:
No, since aXe is a sort of brand name. For example, one wouldn't capitalize the m in macOS but they would the w in Windows, because that's how those brands are identified.

Reply:
Sorry, I mean in `a11y`. I agree with `aXe`.

Reply:
I think it's ok to go either way but personally I'd go with a11y in lowercase, even in a title, as it's more common.

Reply:
Anyway, it is updated as you requested.